### DISCUSSION TITLE
upload finishes and restarts from scratch

### DISCUSSION THREAD

**Discussion by jonkdigit** [iterate-ch/cyberduck#14729] [Q&A]
I just spent day&night to upload 199Gb file when finished at 100%, it restarts at 0%. No artifact of the previous upload can be found. 
In previous issue thread, one mentioned to solve it by setting pasv to post mode, but I cannot find this setting anywhere in Cyberduck. 
I cannot use Cyberduck as long as this issue retains.
anyone know the solution?

Resurrecting https://github.com/iterate-ch/cyberduck/issues/3145

**Comment by tatze96** [comment]
The connection mode can be set to active (PORT) or passive (PASV) within the corresponding field in the _More Options_ section of the connection configuration of FTP(S) connections.

If changing the connection mode doesn't resolve the behavior:
Please [enable debug logging](https://docs.cyberduck.io/cyberduck/support/#debug-logging-level), re-create the described behavior, and send us the log file afterward. If you don't like to see sensitive data to be shared publicly, please send the log files with the ticket reference GH-14693 to support@cyberduck.io, otherwise attach it here.

**Sidenote**: If you want to share the log file here, make sure to attach it through the GitHub web interface, as it wouldn't be visible to us otherwise.
